packages/flutter/lib/src/web.dart

```diff
@@ -0,0 +1,200 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A stripped down version of `package:web` to avoid pinning that repo in
+/// Flutter as a dependency.
+///
+/// This should stay in sync with `package:web` as much as possible to make it
+/// easier to add new members as needed.
```

Comment:
What is the process for keeping it in sync with `package:web`? Is this file generated? If so, we should document the process for regenerating it. If it's manually written, we should document the guidelines for maintaining it.

Reply:
It is indeed manually copied over. I've added a few comments guiding users on how to maintain and extend it, let me know if there needs to be more, thanks!